t/ui/18-tests-details.t

```diff
@@ -47,6 +47,12 @@ unless ($driver) {
     exit(0);
 }
 
+sub disable_bootstrap_fade_animation {
```

Comment:
In Phantom tests it is always required to disable any kind of animation if we want to get rid of sleeps.